### PR TITLE
Error handling with APIException

### PR DESCRIPTION
**Pull request by andrzejstopka**

### CODE REVIEW

File: backend/utility_functions/api_exceptions.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from flask_api.exceptions import APIException
+
+
+class InternalServerError(APIException):
+    detail = "Internal Server Error"
+    status_code = 500
+
+class BadRequest(APIException):
+    detail = None
+    status_code = 404
```
Comment (Hankier) [inline backend/utility_functions/api_exceptions.py:10]: Change to 400

File: backend/requirements.txt [hunk 2]
```diff
@@ -3,6 +3,7 @@ charset-normalizer==3.0.1
 click==8.1.3
 colorama==0.4.6
 Flask==2.2.2
+flask_api = 3.0.post1
```
Comment (Hankier) [inline backend/requirements.txt:6]: Is that proper format?

**Review by Hankier — CHANGES_REQUESTED** [review]
Change status code and check requiremnts file format.

File: backend/utility_functions/api_exceptions.py (hunk 1 above)
Comment (andrzejstopka) [reply backend/utility_functions/api_exceptions.py]: 4b38fb4e01bb4e3742f60b1891f7fd85a616f477 Done

File: backend/requirements.txt (hunk 2 above)
Comment (andrzejstopka) [reply backend/requirements.txt]: 4b38fb4e01bb4e3742f60b1891f7fd85a616f477 Done

**Review by Hankier — APPROVED** [review]

**Pull request merged**